lib/ansible/modules/system/pamd.py

```diff
@@ -248,7 +248,7 @@
     description:
     - "That action that was taken and is one of: update_rule,
       insert_before_rule, insert_after_rule, args_present, args_absent,
-      absent."
+      absent. This was available in Ansible version 2.4 and removed in 2.7"
```

Comment:
Was this previously removed and not documented as such?

Reply:
I removed `action` from the return values in this change since the action taken is already available in the invocation details of the module. It is superfluous. I can add it back into the return values if desired.

Reply:
```suggestion
      absent. This was available in Ansible version 2.4 and removed in 2.8"
```
We can change it to 2.7 or 2.6 when it gets backported.